src/D2L.CodeStyle.Analyzers/Immutability/ImmutableDefinitionChecker.cs

```diff
@@ -249,33 +339,29 @@ ExpressionSyntax initializer
 		/// </summary>
 		/// <param name="memberType">The type of the field/property</param>
 		/// <param name="memberTypeSyntax">The syntax for the type of the field/property</param>
-		/// <param name="initializer">The initializer syntax for the field/property (possibly null)</param>
+		/// <param name="assignment">The assignment syntax for the field/property (possibly null)</param>
 		/// <returns>null if no checks are needed, otherwise a bunch of stuff.</returns>
 		private (ITypeSymbol, ImmutableTypeKind, Func<Location>)? GetStuffToCheckForMember(
 			ITypeSymbol memberType,
 			TypeSyntax memberTypeSyntax,
-			ExpressionSyntax initializer
+			ExpressionSyntax assignment

```

Comment:
This one actually is the initializer :)

Looking at this method in isolation, if I were to call it, I'd be wondering "which assignment"?
When it's called `initializer` I know exactly what it means and how to call it.

Reply:
It's actually all of them. This method gets called for each assignment, not just the initializer. See `CheckFieldOrProperty(...)`.

Reply:
Ohh yes good stuff.